app/controllers/posts_controller.rb

```diff
@@ -8,4 +8,27 @@ def show
     @user = User.find(params[:user_id])
     @post = Post.find(params[:id])
   end
+
+  def new
+    post = current_user.posts.new
+    render :new, locals: { post: post }
+  end
+
+  def create
+    post = current_user.posts.new(params.require(:post).permit(:title, :text))
+    @user = current_user
+    @posts = @user.posts
```

Comment:
- [ ] Please make use of strong params. see [reference](https://guides.rubyonrails.org/action_controller_overview.html#strong-parameters)